src/components/Reports/ReportsBox/ReportsBox.js

```diff
@@ -87,9 +87,8 @@ class ReportsBox extends React.Component {
                 <FormattedMessage id="reports_box.visits_with_email" />
               </h6>
               <small className="date-range">
-                <FormattedDate value={dateFrom} timeZone="UTC" />{' '}
-                <FormattedMessage id="reports_box.to" />{' '}
-                <FormattedDate value={dateTo} timeZone="UTC" />
+                <FormattedDate value={dateFrom} /> <FormattedMessage id="reports_box.to" />{' '}
```

Comment:
It is out of the scope of this PR, but I would like to comment it for the future, there should not be a text entry for `to`, there should be something like `{dateFrom} to {dateTo}`.